.travis.yml

```diff
@@ -1,21 +1,22 @@
 language: node_js
-node_js: 8
 git:
   depth: false
-branches:
-  only:
-    - master
 
 dist: trusty
 sudo: required
 
-env:
-  - TEST_SUITE=screenshots
-  - TEST_SUITE=units
+matrix:
+  include:
+    - node_js: 8
+      env: TEST_SUITE=unit
```

Comment:
I don't, but they help to provide names for the builds in Travis. Otherwise, you'd have to click into a build and parse the logs to figure out which test it is.

<img width="1343" alt="screen shot 2018-03-19 at 10 10 35 am" src="https://user-images.githubusercontent.com/5939574/37610799-d42373ae-2b5d-11e8-9244-531cc282cf34.png">